packages/manager/apps/container/src/container/legacy/server-sidebar/universe/HostedPrivateCloudSidebar.tsx

```diff
@@ -236,6 +241,46 @@ export default function HostedPrivateCloudSidebar() {
         ],
       });
     }
+    if (feature['veeam-backup'] || feature['hycu']) {
+      menu.push({
+        id: 'hpc-storage-backup',
+        label: t('sidebar_storage_backup'),
+        icon: <img className="mb-1 mr-1 w-6 aspect-square" alt="" src={infinityCLoud} />,
+        pathMatcher: new RegExp('(^/veeam-backup)|(^/hycu)'),
+        badge: 'new',
+        ignoreSearch: true,
+        subItems: [
+          // TODO : Insert veeam-backup submenu here
+          // Wait #12611
+          // See : https://github.com/ovh/manager/pull/12611/files#diff-d0f2536b9e46dd58ef3f64f2e1b51320388020cc137b40f6f7f9241f83556f47
+          (feature['hycu']) && {
+            id: 'hpc-hycu',
+            label: t('sidebar_hycu'),
+            icon: <img alt="" src={hycuLogo} className="mb-1 w-6 aspect-square" />,
+            pathMatcher: new RegExp('^/hycu'),
+            badge: "new",
+            async loader() {
+              const appId = 'hycu';
+              const items = await loadServices('/hycu');
+
+              return [
+                {
+                  id: 'hycu-all',
+                  label: t('sidebar_all_hycu'),
+                  href: navigation.getURL(appId, '#/'),
+                  ignoreSearch: true,
+                },
+                ...items.map((service) => ({
+                  ...service,
+                  href: navigation.getURL('dedicated', `#/hycu/${service.serviceName}`),
```

Comment:
**nitpick:** use the appId variable
l.264 : ```loadServices(`/${appId}`)```
l.275 : ```navigation.getURL('dedicated', `#/${appId}/${service.serviceName}`)```

Reply:
Done, Thank you :)